docs/source/models/concept_models/vivarium_nutrition_optimization/concept_model.rst

```diff
@@ -57,14 +57,77 @@ Documents that contain information specific to the subcomponents of this simulat
 2.0 Modeling aims and objectives
 ++++++++++++++++++++++++++++++++
 
+**Objective:** For a given fiscal envelope, determine the optimal funding allocation across select nutrition-related intervention products for maternal and child health. "Optimal" is defined with respect to some specified outcome such as the funding allocation that minimizes DALYs or maximizes the number of alive, non-stunted children.
 
+.. note::
+  
+  We must be careful in optimizing for "minimizing" the prevalence of some risk such as child growth failure. This is because prevalence of such risks could be decreased via decreasing incidence (positive health impact) *or* via increasing risk-related mortality (negative health impact). That is why we choose to *maximize* the number of alive non-stunted children rather than minimize the prevalence of child growth failure.
+
+2.1 General modeling strategy
+-----------------------------
+
+2.1.1 Conceptualizing "mixed" intervention coverage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To achieve the general modeling objective of determining the most cost effective combination of interventions for a given budget size. When we think about "*combinations* of interventions," it is important to distinguish between two modes in which coverage of more than one intervention may occur. For example, imagine a scenario in which we have 50% population coverage of intervention A and 50% population coverage of intervention B. Any of the following could be true:
+
+1. Combined intervention coverage among the same individuals: 50% of the population is covered by intervention A and B, 50% of the population is covered by neither intervention A nor B
+
+2. Combined intervention coverage among separate individuals: 50% of the population is covered by intervention A, 50% of the population is covered by intervention B
+
+3. Some combination of 1 and 2
+
+
+**In the case of the first situation:**
+
+  There may be important interactions between the *simultaneous* coverage of intervention A and B in the same subpopulation. For example:
+
+  - An infant who was covered by antenatal supplementation may survive the neonatal period due to associated improvements to their LBWSG exposure and therefore survive to benefit from SQLNS supplementation later in life
+  - A child who may have benefitted from SAM treatment alone may no longer need to be treated for SAM if they are covered by MAM treatment as well as SAM treatment
```

Comment:
Does this complicate the calculation of cost at all? My understanding is that we have defined "coverage" such that this child is "covered" by SAM treatment, regardless of whether or not they actually need it. But I'd assume that this "coverage" is cheaper when the treatment doesn't actually need to be performed.

Reply:
Coverage is really "probability you will receive treatment if you are eligible/need it." All our simulants are subject to that same probability regardless of whether they need the treatment. But yeah, our simulation outputs both the number of treated SAM cases and total SAM cases (which actually will be the same because we'll have 100% coverage), for instance, so we calculate costs directly from how many treatments were actually administered rather than based on the cost of "100% coverage"

Reply:
Makes sense. I guess I didn't realize until now that cost requires an "emulator" just as complex as impact, because the cost of 100% coverage of A and B is not the same as the cost of 100% coverage of A + the cost of 100% coverage of B.

Reply:
![image](https://github.com/ihmeuw/vivarium_research/assets/54918472/e5744b8a-2119-4b12-abfa-d5e94bcb8f0f)
Yeah! That's how we get this cool trend where the cost of full coverage of SAM (and MAM) treatment decreases as coverage of other interventions increases

Reply:
Hmm yeah, good question. The way this is typically handled in other models is that there is a price for an "intervention bundle" if the bundle is just SAM tx, then cost will be screening + the cost of SAM tx. If the bundle is SAM and MAM tx, then cost is screening + SAM tx + MAM tx and the screening cost is not specific to either. This is likely how we will treat it in our scenarios/emulator, but that doesn't answer how we should present it in these sorts of figures.

I suppose we could either (1) spread out screening (or other sorts of shared costs) across all the interventions they apply to equally, or (2) have a separate color in the bar chart for screening costs. I like the second option better.